### ISSUE TITLE
More/better CSS classes in markup

### ISSUE THREAD

**Issue by chrisgherbert** [firesidelabs/fireside-features#106]
I've been using CSS to tweak some aspects of our site, but the classes on elements aren't very granular.  Some suggestions:

- include a class on the body element indicating which template is being using (eg, home, about, single-episode, etc).

- Even when elements are styled the same, add classes to distinguish them.  For example, on episode pages the "Host" and "Guest" thumbnail look the same, but it might be helpful to target them separately.  Maybe a BEM-style class of `.episode-hosts--hosts` and `.episode-hosts--guests`?

**Comment by ChrisOOS** [comment]
Agreed. Ability to individualize things like the content of the h2 for each page would be great touch. All in favor of things that help individualize podcasts

**Comment by dan** [comment]
I've assigned this to our designer.

**Comment by Glark** [comment]
I was coming here to suggest a body class too, thank you. The solution should include custom pages as well e.g. `.custom-user-submissions`.
